deepchem/models/tests/test_infograph.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import numpy as np
+import deepchem as dc
+from deepchem.feat.molecule_featurizers import MolGraphConvFeaturizer
+from deepchem.data import NumpyDataset
+from deepchem.models.torch_models.infograph import InfoGraphModel
+from deepchem.molnet import load_bace_classification, load_delaney
+try:
+    import torch
+except:
+    pass
+
+
+@pytest.mark.torch
+def get_dataset(mode='classification', num_tasks=1):
+    np.random.seed(123)
+    data_points = 20
+    featurizer = MolGraphConvFeaturizer(use_edges=True)
+
+    if mode == 'classification':
+        tasks, all_dataset, transformers = load_bace_classification(featurizer)
+    else:
+        tasks, all_dataset, transformers = load_delaney(featurizer)
```

Comment:
This will load the full dataset which may be slow. Perhaps better to make a tiny dataset of 20 compounds directly for testing (the data_points specification below only avoids training on the full set I think and doesn't avoid the load)

Reply:
I can change it, but these functions are used in about a dozen tests. Maybe would be better to a create a load function specifically for testing.